catalog/files/impl/src/main/java/org/projectnessie/catalog/files/gcs/GcsOptions.java

```diff
@@ -92,61 +92,36 @@ public interface GcsOptions<PER_BUCKET extends GcsBucketOptions> {
   @ConfigItem(section = "transport")
   OptionalDouble rpcTimeoutMultiplier();
 
-  /** The read chunk size in bytes. */
-  @ConfigItem(section = "transport")
-  OptionalInt readChunkSize();
```

Comment:
These 3 options were misplaced, their right place is in `GcsBucketOptions` where they are also declared.